MAUI/Button/Customization.md

```diff
@@ -0,0 +1,671 @@
+            Stroke="Black"
+            Background="White"
+            ImageAlignment="Bottom"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+SfButton button = new SfButton()
+{
+    Text = "Shopping",
+    TextColor = Color.Black,
+    HorizontalOptions = LayoutOptions.Center,
+    ImageSource = "add_to_card.png",
+    ShowIcon = true,
+    ImageSize = 25,
+    Stroke = Colors.Black,
+    Background = Colors.White,
+    ImageAlignment = Alignment.Bottom
+};
+
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with icon image with top alignment](images/Button_imagealignment_bottom.png)
+
+**Left image alignment in `SfButton`**
+
+In RTL flow direction, image alignment with `Start` will change its direction of placing image to the right. To keep that in same left position, set `Left` alignment as shown in the following code sample.
+
+{% tabs %}
+{% highlight xaml %}
+
+      <buttons:SfButton  
+            Text="Shopping"
+            TextColor="Black"
+            HorizontalOptions="Center"
+            ImageSource="add_to_card.png"
+            ShowIcon="True" 
+            ImageSize="25"
+            Stroke="Black"
+            Background="White"
+            ImageAlignment="Left"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfButton button = new SfButton()
+{
+    Text = "Shopping",
+    TextColor = Color.Black,
+    HorizontalOptions = LayoutOptions.Center,
+    ImageSource = "add_to_card.png",
+    ShowIcon = true,
+    ImageSize = 25,
+    Stroke= Colors.Black,
+    Background = Colors.White,
+    ImageAlignment = Alignment.Left
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with icon image with left alignment](images/Button_imagealignment_left.png)
+
+**Right image alignment in `SfButton`**
+
+In RTL flow direction, image alignment with `End` will change its direction of placing image to the left. To keep that in same right position, set `Right` alignment as shown in the following code sample.
+
+{% tabs %}
+{% highlight xaml %}
+
+      <buttons:SfButton  
+            Text="Shopping"
+            TextColor="Black"
+            HorizontalOptions="Center"
+            ImageSource="add_to_card.png"
+            ShowIcon="True" 
+            ImageSize="25"
+            Stroke="Black"
+            Background="White"
+            ImageAlignment="Right"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+SfButton button = new SfButton()
+{
+    Text = "Shopping",
+    TextColor = Color.Black,
+    HorizontalOptions = LayoutOptions.Center,
+    ImageSource = "add_to_card.png",
+    ShowIcon = true,
+    ImageSize = 25,
+    Stroke= Colors.Black,
+    Background = Colors.White,
+    ImageAlignment = Alignment.Right
+};
+
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfButton with image with icon image with right alignment](images/Button_imagealignment_right.png)
+
+## Gradient background
+
+You can set the gradient as background of SfButton using the `Background` property. It supports the following types of gradients:
```

Comment:
You can set the gradient as the background of SfButton using the `Background` property. It supports the following types of gradients: